packages/server/graphql/public/types/User.ts

```diff
@@ -85,6 +85,12 @@ const User: UserResolvers = {
     }
     return request
   },
+  favoriteTemplates: async ({favoriteTemplateIds}, _args, {dataLoader}) => {
+    const templates = await Promise.all(
+      favoriteTemplateIds.map((templateId) => dataLoader.get('meetingTemplates').load(templateId))
```

Comment:
-1 Why not use `dataLoader.get('meetingTemplates').loadMany(favouriteTemplateIds).filter(isValid)`?